packages/ihe-gateway/.env.example

```diff
@@ -24,7 +24,15 @@ IHE_GW_FULL_BACKUP_LOCATION="../../../another-folder"
 
 # S3 bucket name for storing the SSL certs and custom extensions - leave empty to skip that
 CONFIG_BUCKET_NAME=""
+S3_BUCKET_NAME=""
 
 # AWS credentials for the S3 bucket
 AWS_ACCESS_KEY_ID=""
 AWS_SECRET_ACCESS_KEY=""
+
+# XCPD and XCA request URLs
+INBOUND_XCPD_URL=""
+INBOUND_XCA38_URL=""
+INBOUND_XCA39_URL=""
```

Comment:
if we're agreed, let's change the naming scheme here to:
```
INBOUND_PATIENT_DISCOVERY_URL=""
INBOUND_DOCUMENT_QUERY_URL=""
INBOUND_DOCUMENT_RETRIEVE_URL=""
```

going back and forth everywhere between XCA/XCPD and our semantic terminology is confusing and not consistent - how about all new code uses our PD/DQ/DR scheme?

this will also address @leite08's comment here https://github.com/metriport/metriport/pull/1521/files#r1482351731